### ISSUE TITLE
[Bug]: can not start Server (NeoForge 1.20.4)

### ISSUE THREAD

**Issue by MSandro** [Griefed/ServerPackCreator#516]
### What happened?

I have used the ServerPackCreator to create the Server File for my NeoForge 1.20.4 Modpack.
after starting the start.sh file it says something went wrong.

### What did you expect to happen?

it should install the server.

### Version

5.1.18

### Relevant log output

```shell
Start script generated by ServerPackCreator 5.1.18.
To change the launch settings of this server, such as JVM args / flags, Minecraft version, modloader version etc., edit the variables.txt-file.

Running NeoForge checks and setup...
Generating user_jvm_args.txt from variables...
Edit JAVA_ARGS in your variables.txt. Do not edit user_jvm_args.txt directly!
Manually made changes to user_jvm_args.txt will be lost in the nether!
libraries/net/neoforged/forge/1.20.4-20.4.211/forge-1.20.4-20.4.211-server.jar could not be found.
Downloading neoforge-installer.jar
from https://maven.neoforged.net/net/neoforged/forge/1.20.4-20.4.211/forge-1.20.4-20.4.211-installer.jar

Download complete.
NeoForge Installer downloaded. Installing...
Error: Invalid or corrupt jarfile neoforge-installer.jar
Something went wrong during the server installation. Please try again in a couple of minutes and check your internet connection.
Exiting...
Press any key to continue
```

- [X] I agree to follow this project's Code of Conduct

**Comment by MSandro** [comment]
i was able to fix the link by my own, but I'll keep this issue open.

**Comment by MSandro** [comment]
Now I get this error:
```
  ...
  net/minecraft/world/ticks/TickPriority.class
  net/minecraft/world/ticks/WorldGenTickAccess.class
  net/minecraft/world/ticks/package-info.class
===============================================================================
Processor: net.neoforged.installertools:binarypatcher
  MainClass: net.neoforged.binarypatcher.ConsoleTool
  Classpath:
    /home/sandro/Downloads/MC/server/test/./libraries/net/neoforged/installertools/binarypatcher/2.1.2/binarypatcher-2.1.2.jar
    /home/sandro/Downloads/MC/server/test/./libraries/net/sf/jopt-simple/jopt-simple/5.0.4/jopt-simple-5.0.4.jar
    /home/sandro/Downloads/MC/server/test/./libraries/net/neoforged/installertools/binarypatcher/2.1.2/binarypatcher-2.1.2.jar
    /home/sandro/Downloads/MC/server/test/./libraries/trove/trove/1.0.2/trove-1.0.2.jar
    /home/sandro/Downloads/MC/server/test/./libraries/net/neoforged/installertools/cli-utils/2.1.2/cli-utils-2.1.2.jar
    /home/sandro/Downloads/MC/server/test/./libraries/net/neoforged/srgutils/1.0.0/srgutils-1.0.0.jar
    /home/sandro/Downloads/MC/server/test/./libraries/com/github/jponge/lzma-java/1.3/lzma-java-1.3.jar
    /home/sandro/Downloads/MC/server/test/./libraries/com/nothome/javaxdelta/2.0.1/javaxdelta-2.0.1.jar
  Args: --clean, /home/sandro/Downloads/MC/server/test/./libraries/net/minecraft/server/1.20.4-20231207.154220/server-1.20.4-20231207.154220-srg.jar, --output, /home/sandro/Downloads/MC/server/test/./libraries/net/neoforged/neoforge/20.4.211/neoforge-20.4.211-server.jar, --apply, /tmp/forge_installer8542948289178809783/data/server.lzma
Applying: 
  Clean:     /home/sandro/Downloads/MC/server/test/./libraries/net/minecraft/server/1.20.4-20231207.154220/server-1.20.4-20231207.154220-srg.jar
  Output:    /home/sandro/Downloads/MC/server/test/./libraries/net/neoforged/neoforge/20.4.211/neoforge-20.4.211-server.jar
  KeepData:  false
  Unpatched: false
  Pack200:   false
  Legacy:    false
Loading patches file: /tmp/forge_installer8542948289178809783/data/server.lzma
Processor: net.neoforged.installertools:binarypatcher: Loading patch files
Processor: net.neoforged.installertools:binarypatcher: Patching input
Processor: net.neoforged.installertools:binarypatcher: Adding new files
The server installed successfully
You can delete this installer file now if you wish
Something went wrong during the server installation. Please try again in a couple of minutes and check your internet connection.
Exiting...
```

**Comment by MSandro** [comment]
I have edited the NeoForge URL because it was not able to download.
Here is the log with the original Script:

`Start script generated by ServerPackCreator 5.1.18.
To change the launch settings of this server, such as JVM args / flags, Minecraft version, modloader version etc., edit the variables.txt-file.

Running NeoForge checks and setup...
Generating user_jvm_args.txt from variables...
Edit JAVA_ARGS in your variables.txt. Do not edit user_jvm_args.txt directly!
Manually made changes to user_jvm_args.txt will be lost in the nether!
libraries/net/neoforged/forge/1.20.4-20.4.211/forge-1.20.4-20.4.211-server.jar could not be found.
Downloading neoforge-installer.jar
from https://maven.neoforged.net/net/neoforged/forge/1.20.4-20.4.211/forge-1.20.4-20.4.211-installer.jar

Download complete.
NeoForge Installer downloaded. Installing...
Error: Invalid or corrupt jarfile neoforge-installer.jar
Something went wrong during the server installation. Please try again in a couple of minutes and check your internet connection.
Exiting...
Press any key to continue`

**Comment by Griefed** [comment]
I'll take a look at NeoForge 1.20.x this week.

**Comment by Griefed** [comment]
Version 5.1.19 will be released shortly. The links for NeoForge have been moved to the variables.txt to accomodate for the different URLs which NeoForge uses depending on the Minecraft version used (1.20.1 vs 1.20.2 and up)

**Comment by Griefed** [comment]
Fixed in [5.1.19](https://github.com/Griefed/ServerPackCreator/releases/tag/5.1.19)
